simplekv/net/botostore.py

```diff
@@ -87,10 +88,41 @@ def _get_filename(self, key, filename):
             return k.get_contents_to_filename(filename)
 
     def _open(self, key):
+        from boto.s3.keyfile import KeyFile
+
+        class SimpleKeyFile(KeyFile):
+            def read(self, size=-1):
+                if self.closed:
+                    raise ValueError("I/O operation on closed file")
+                if size < 0:
+                    size = self.key.size - self.location
+                return KeyFile.read(self, size)
+
+            def seekable(self):
```

Comment:
What is the purpose of these functions?

Reply:
See https://docs.python.org/3/library/io.html#io.IOBase.seekable

Reply:
👍